File: generators/java/src/main/java/dqualizer/dqlang/archive/loadtesttranslator/dqlang/modeling/RuntimeQualityAnalysis.java
package dqualizer.dqlang.archive.loadtesttranslator.dqlang.modeling;

import lombok.Getter;
import lombok.ToString;

import java.util.LinkedHashSet;

@Getter
@ToString
public class RuntimeQualityAnalysis {

    private LinkedHashSet<Object> monitoring;
    private LinkedHashSet<Object> resilience;
    private LinkedHashSet<ModeledLoadTest> loadtests;
}

File: src/main/kotlin/dqualizer/dqlang/archive/loadtesttranslator/dqlang/modeling/RuntimeQualityAnalysis.kt
package dqualizer.dqlang.archive.loadtesttranslator.dqlang.modeling

import lombok.Getter
import lombok.ToString

@Getter
@ToString
class RuntimeQualityAnalysis {
    private val monitoring: LinkedHashSet<Any>? = null
    private val resilience: LinkedHashSet<Any>? = null
    private val loadtests: LinkedHashSet<ModeledLoadTest>? = null
}
